core/migrations/1666463689317-Automigration.ts

```diff
@@ -0,0 +1,94 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class Automigration1666463689317 implements MigrationInterface {
+    name = 'Automigration1666463689317'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "user" DROP COLUMN "type"`);
+        await queryRunner.query(`ALTER TABLE "history"."user_history" DROP COLUMN "type"`);
```

Comment:
Why are we touching history tables here?

Reply:
Type is irrelevant and was never used.

Reply:
It was something Nigel added in long ago but we ended up using MLE Org Teams.

Reply:
We still shouldn't drop columns from history tables regardless